src/cpu/arm.rs

```diff
@@ -286,24 +281,8 @@ impl Feature {
         if self.mask == self.mask & ARMCAP_STATIC {
             return true;
         }
-
-        #[cfg(all(
-            any(
-                target_os = "android",
-                target_os = "fuchsia",
-                all(target_os = "linux", not(target_env = "uclibc")),
-                target_os = "windows"
-            ),
-            any(target_arch = "arm", target_arch = "aarch64")
-        ))]
-        {
-            // SAFETY: See `OPENSSL_armcap_P`'s safety documentation.
-            if self.mask == self.mask & unsafe { OPENSSL_armcap_P } {
-                return true;
-            }
-        }
-
-        false
+        // SAFETY: See `OPENSSL_armcap_P`'s safety documentation.
+        self.mask == self.mask & unsafe { OPENSSL_armcap_P }
```

Comment:
This function isn't currently used by aarch64 targets for anything other than NEON; all the other checks of `OPENSSL_armcap_P` are done in assembly code for AArch64.

This line covered by 32-bit ARM targets in the `test` matrix but it isn't covered in the `coverage` matrix because 32-bit ARM targets don't support source-based code coverage measurement yet.